app/controllers/catalog_controller.rb

```diff
@@ -83,6 +85,7 @@ class CatalogController < ApplicationController
     # config.show.thumbnail_field = 'thumbnail_path_ss'
     config.show.document_presenter_class = Arclight::ShowPresenter
     config.show.metadata_partials = %i[
+      using_these_materials_field
```

Comment:
It feels a little odd to put this in with metadata partials since we're just displaying a list of links. But I guess the other way of doing this would involve overriding the DocumentComponent template.

Reply:
I think this works for now. I could see us wanting to override and move it to the DocumentComponent partial if we want to fuss with the layout more in the future.